### PR TITLE
fix(#2173): consider variables in minUI5Version

### PR DESCRIPTION
**Pull request by hitesh-parmar**
Follow up to issue: #2173 

semver valid check at getMinUI5VersionAsArray was introduced with PR https://github.com/SAP/open-ux-tools/pull/2175 but this causes problems with applications using `variables` as minUI5Version. This PR fixes that.

Before this PR

Application Info Page
image

After this PR
Application Info Page
image

### CODE REVIEW

File: packages/project-access/test/project/minUI5Version.test.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ describe('getters for minUI5Version', () => {
     describe('getMinUI5VersionFromManifest', () => {
         const testVersions: Array<TestCase> = [
             { minUI5Version: undefined, expectedResult: undefined },
+            { minUI5Version: '${sap.ui5.dist.veresion}', expectedResult: '${sap.ui5.dist.veresion}' },
```
Comment (devinea) [inline packages/project-access/test/project/minUI5Version.test.ts:12]:
```suggestion
            { minUI5Version: '${sap.ui5.dist.version}', expectedResult: '${sap.ui5.dist.version}' },
```
Typo? Same below.

Comment (hitesh-parmar) [reply packages/project-access/test/project/minUI5Version.test.ts]: Thanks for the review @devinea . Typo corrected with next commit

**Review by devinea — APPROVED** [review]
Code changes are clear and covered by tests. 
Changeset lg.

**Review by AlinaGovoruhina — APPROVED** [review]
App info page is updated, code is covered by tests

**Review by 815are — APPROVED** [review]
checked:
1. change is straightforward and simple to understand 
2. test is updated with additional cases
3. changeset persists

**Comment by Klaus-Keller** [comment]
@hitesh-parmar, can I veto this change. To see also invalid versions like `${sap.ui5.dist.version}` consumers can call the function `getMinUI5VersionAsArray()` with optional parameter `noValidation = true`.

**Comment by hitesh-parmar** [comment]
Thanks for the review @Klaus-Keller. I will revert this change as discussed.
The reason for this solution is that I consider `${sap.ui5.dist.version}` as a valid version in the `manifest.json`. 

With the previous change valid versions are only those that can be loaded from CDN https://ui5.sap.com/

**Pull request merged**